cs/configuring.texy

```diff
@@ -107,6 +108,28 @@ $configurator->addParameters([
 ]);
 \--
 
+Lze definovat i dynamické parametry, které nezpůsobí přegenerování containeru.
+
+/--php
+$configurator->addDynamicParameters([
+	'rootDir' => 'foobar',
+]);
+\--
+
+
+Přidání služeb
+--------------
+
+Služby můžeme do DI containeru přidat i ručně.
```

Comment:
Tedka jsi me trochu zaskocil, tzn, ze ja v neonu definuju tridu uplne normalne jak jsem zvyklej, ale nastavim ji `dynamic: yes`, v tom pripade ji Nette ocekava, ze ji pridam pres addServices jo? Wuf, to jsem vubec nevedel. Vyhodi se vyjimka, pokud ji nedostane?